app/main.py

```diff
@@ -1,8 +1,42 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int, clean_mark: int, brand: str):
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self, distance_from_city_center: float,
+                 clean_power: int, average_rating: float,
+                 count_of_ratings: int):
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, cars: list):
+        cost = 0
+        # calculate total cost of washing all cars
+        for car in cars:
+            if self.clean_power > car.clean_mark:
+                cost += self.calculate_washing_price(car)
+                self.wash_single_car(car)
+        return round(cost, 1)
+
+    def calculate_washing_price(self, car):
```

Comment:
Add annotations everywhere, and for return values as well, for example:
```
def rate_service(self, rating: float) -> None:
```

Reply:
> Add annotations everywhere, and for return values as well, for example:
> 
> ```
> def rate_service(self, rating: float) -> None:
> ```

Done.